github_activity/cli.py

```diff
@@ -1,49 +1,67 @@
 import argparse
-import sys
 import os
-import os.path as op
+import sys
+
 from .github_activity import generate_activity_md
 
 DESCRIPTION = "Generate a markdown changelog of GitHub activity within a date window."
 parser = argparse.ArgumentParser(description=DESCRIPTION)
 parser.add_argument(
-    "target", help="The GitHub organization/repo for which you want to grab recent issues/PRs."
-                   "Can either be *just* and organization (e.g., `jupyter`) or a combination"
-                   "organization and repo (e.g., `jupyter/notebook`). If the former, all"
-                   "repositories for that org will be used. If the latter, only the specified"
-                   "repository will be used.")
+    "target",
+    help="""The GitHub organization/repo for which you want to grab recent issues/PRs.
+    Can either be *just* and organization (e.g., `jupyter`) or a combination
+    organization and repo (e.g., `jupyter/notebook`). If the former, all
+    repositories for that org will be used. If the latter, only the specified
+    repository will be used.""",
+)
+parser.add_argument(
+    "since",
+    help="""Return issues/PRs with activity since this date or git reference. Can be
+    any string that is parsed with dateutil.parser.parse.""",
+)
+parser.add_argument(
+    "-o", "--output", default=None, help="""Write the markdown to a file if desired."""
+)
 parser.add_argument(
-    "since", help="Return issues/PRs with activity since this date. Can "
-                    "be any string that is parsed with pd.to_datetime.")
-parser.add_argument("-o", "--output", default=None,
-                    help="Write the markdown to a file if desired.")
-parser.add_argument("--before", default=None,
-                    help="Return issues/PRs with activity before this date. "
-                            "Can be any string that is parsed with pd.to_datetime. "
-                            "If none, today's date will be used.")
-parser.add_argument("--repo", default=None,
-                    help="A GitHub repository for which you want to grab recent issues/PRs. "
-                            "If None, all repositories for `org` will be used.")
-parser.add_argument("--kind", default=None,
-                    help="Return only issues or PRs. If None, both will be returned.")
+    "--until",
+    default=None,
+    help="""Return issues/PRs with activity until this date or git reference. Can be
+    any string that is parsed with dateutil.parser.parse. If none, today's
+    date will be used.""",
+)
+parser.add_argument(
+    "--repo",
```

Comment:
I'm thinking we should just remove this argument since we tell people to do either `org` or `org/repo` in the `target` argument. What do you think?

Reply:
Yepp, especially since `args.repo` was never used! I did that in 907bd3a pushed now.